static/sass/styles.scss

```diff
@@ -246,6 +246,11 @@ select {
   margin-bottom: 1.25rem;
 }
 
+.p-fieldset-section {
```

Comment:
This can be removed once the feature branch rebases from main, as this was added in the refresh forms feature branch